bot/utils/login_prompt.py

```diff
@@ -0,0 +1,14 @@
+from telegram import Update
+
+from bot.models import User
+
+def prompt_login_check(update: Update, user: User) -> bool:
+    if not user or not user.spotify:
```

Comment:
consider reversing check logic to return True in the indented block so you can send answer on the top level of the function

```python
if not (user and user.spotify):
    return True
answer()
return False
```